ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -24,10 +27,25 @@ def Acosh(*, x, name="Acosh"):
     return ivy.acosh(x)
 
 
-Add = tf_frontend.math.add
+@to_ivy_arrays_and_back
+def Add(*, x, y, name="Add"):
```

Comment:
In this implementation, we will wrap the function every time we call `Add` which is not good. We should apply the decorator only once at import time. It would be better if we did something like:

```
Add = map_raw_ops_alias(tf_frontend.math.add, x=x, y=y, name=name)
```

This will only be called at import time, and will make `Add` much more runtime efficient.